tests/sqlparser_hive.rs

```diff
@@ -32,6 +32,20 @@ fn parse_table_create() {
     hive().verified_stmt(iof);
 }
 
+fn generic(options: Option<ParserOptions>) -> TestedDialects {
+    TestedDialects {
+        dialects: vec![Box::new(GenericDialect {})],
+        options,
+    }
+}
+
+#[test]
+fn parse_describe() {
+    let describe = r#"DESCRIBE namespace.`table`"#;
```

Comment:
this I think was part of https://github.com/sqlparser-rs/sqlparser-rs/pull/1010, right?